Samples/outlook-tag-external/manifest.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<OfficeApp xmlns="http://schemas.microsoft.com/office/appforoffice/1.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:bt="http://schemas.microsoft.com/office/officeappbasictypes/1.0" xmlns:mailappor="http://schemas.microsoft.com/office/mailappversionoverrides/1.0" xsi:type="MailApp">
+  <Id>84cc5da3-2a5e-4373-8130-7c38a2a6eaf3</Id>
+  <Version>1.0.0.0</Version>
+  <ProviderName>Microsoft</ProviderName>
+  <DefaultLocale>en-US</DefaultLocale>
+  <DisplayName DefaultValue="PnP Tag External Recipients sample"/>
+  <Description DefaultValue="Updates the subject and adds a disclaimer if external recipients are included during message compose."/>
+  <IconUrl DefaultValue="https://elizabethsamuel-msft.github.io/PnP-OfficeAddins/Samples/outlook-tag-external/assets/icon-64.png"/>
```

Comment:
Please move the icons into a directory in the PnP-OfficeAddins repo and link to them there.

Reply:
This is correct for now. Once this change is merged and the pages are generated, I'll update these links in a subsequent PR.